tests/test_panther_sdyaml_pipeline.py

```diff
@@ -0,0 +1,28 @@
+import pytest
+from sigma.collection import SigmaCollection
+from sigma.processing.resolver import ProcessingPipelineResolver
+
+# @pytest.fixture

```

Comment:
Will this be implemented after the backend is reviewed and merged?

Reply:
yep, pipelines are going to implemented/tuned as a separate step/ticked. this code is part of initial WIP pr https://github.com/josh-panther/pySigma-backend-panther-wip/

Reply:
yep, pipelines are going to implemented/tuned as a separate step/ticked. this code is part of initial WIP pr https://github.com/josh-panther/pySigma-backend-panther-wip/